src/test/java/seedu/address/logic/parser/NewAddressBookParserTest.java

```diff
@@ -0,0 +1,51 @@
+package seedu.address.logic.parser;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static seedu.address.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.Messages.MESSAGE_UNKNOWN_COMMAND;
+import static seedu.address.testutil.Assert.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.commands.AddAppointmentCommand;
+import seedu.address.logic.commands.AddPatientCommand;
+import seedu.address.logic.commands.HelpCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.appointment.Appointment;
+import seedu.address.model.person.patient.Patient;
+import seedu.address.testutil.AppointmentBuilder;
+import seedu.address.testutil.PatientBuilder;
+import seedu.address.testutil.PersonUtil;
+
+
+public class NewAddressBookParserTest {
+
+    private final NewAddressBookParser parser = new NewAddressBookParser();
+
+    @Test
+    public void parseCommand_addAppointment() throws Exception {
```

Comment:
test functions need to throw exceptions?

Reply:
the existing AddressBookParser test had it so i just followed